xml/System.IO.Compression/BrotliStream.xml

```diff
@@ -551,7 +551,8 @@
         <remarks>
           <para>The `DisposeAsync` method lets you perform a resource-intensive dispose operation without blocking the main thread. This performance consideration is particularly important in a Windows 8.x Store app or desktop app where a time-consuming stream operation can block the UI thread and make your app appear as if it is not working. The async methods are used in conjunction with the <see langword="async" /> and <see langword="await" /> keywords in Visual Basic and C#.</para>
           <para>This method disposes the Brotli stream by writing any changes to the backing store and closing the stream to release resources.</para>
-          <para>Calling `DisposeAsync` allows the resources used by the <see cref="T:System.IO.Compression.BrotliStream" /> to be reallocated for other purposes. For more information, see <see href="/dotnet/standard/garbage-collection/unmanaged">Cleaning Up Unmanaged Resources</see>.</para>
+          <para>Calling `DisposeAsync` allows the resources used by the <see cref="T:System.IO.Compression.BrotliStream" /> to be reallocated for other purposes. For more information, see [Cleaning Up Unmanaged Resources](/dotnet/standard/garbage-collection/unmanaged).</para>
+          Calling `DisposeAsync` allows the resources used by the <see cref="T:System.IO.Compression.BrotliStream" /> to be reallocated for other purposes. For more information, see [Cleaning Up Unmanaged Resources](/dotnet/standard/garbage-collection/unmanaged).
         </remarks>
```

Comment:
So the `<para>` tags mean Markdown links don't render...

<img width="894" height="238" alt="image" src="https://github.com/user-attachments/assets/1a78cb90-bab1-4848-985c-023794938464" />